src/components/TextInput/TextInputLabel/index.js

```diff
@@ -1,32 +1,34 @@
-import React, {PureComponent} from 'react';
+import React, {useRef, useEffect} from 'react';
 import {Animated} from 'react-native';
 import styles from '../../../styles/styles';
 import {propTypes, defaultProps} from './TextInputLabelPropTypes';
 import CONST from '../../../CONST';
 
-class TextInputLabel extends PureComponent {
-    componentDidMount() {
-        if (!this.props.for) {
+function TextInputLabel(props) {
```

Comment:
@kosmydel It is suggested to destructure prop as per the [guideline](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring)

> Always use destructuring to get prop values. Destructuring is necessary to assign default values to props.

Reply:
Also, it's preferred to destructure in constructor than the first line of the component ([slack discussion](https://expensify.slack.com/archives/C01GTK53T8Q/p1689968108162539))

Reply:
Hey, sorry for the late response, I was OOO.